automation/cypress/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Why needed?

Reply:
Not sure if this is just my machine. But I was getting undefined Cypress commands which is fixed by the tsconfig file found in Cypress docs. Happy to remove this if it's breaking something or if there's an alternative to fixing the problem I was getting.

Reply:
Nope happy as long as there is a reason.